### PR TITLE
Pop address

### PR DESCRIPTION
**Pull request by hard75**

### CODE REVIEW

**Review by alexjavabraz — APPROVED** [review]

**Comment by alexjavabraz** [comment]
LGTM

File: src/components/pegout/AddressDialog.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  <v-dialog v-model="showAddressDialog" width="520" persistent>
+    <v-card class="container dialog">
+      <v-col cols="12" align-self="center" class="pt-0">
+        <v-row class="mx-0 mt-3 mb-3 d-flex justify-center">
+            <p class="pop-up-title">Sign this message</p>
+        </v-row>
+        <v-row class="mx-0 mb-10 justify-center">
+            <p class="pop-up-subtitle mb-2">
+              We will the signature to device your Bitcoin address.
+            </p>
```
Comment (ronaldsg20) [inline src/components/pegout/AddressDialog.vue:11]: Fix the typo : `We will use the signature to derive your Bitcoin address.`

File: src/components/pegout/AddressDialog.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+} from 'vue-property-decorator';
+
+@Component
+export default class AddressDialog extends Vue {
+  @Prop() showAddressDialog!: boolean;
+
+  toSign() {
+    this.$emit('toSign');
+  }
+
```
Comment (ronaldsg20) [inline src/components/pegout/AddressDialog.vue:46]: We are using the decorator `@Emit('event')` syntax.

File: src/components/pegout/PegOutForm.vue [hunk 3]
```diff
@@ -62,8 +69,25 @@ import RskFeeSelect from '@/components/pegout/RskFeeSelect.vue';
...
+
+  @Emit()
+  closeAddressDialog() {
+    this.showAddressDialog = false;
+  }
+
+  signTransaction() {
+    this.recipientAddress = '0xtesttesttesttesttest';
+    this.showAddressDialog = false;
+  }
```
Comment (ronaldsg20) [inline src/components/pegout/PegOutForm.vue:86]: Change the name to `signMessage`,  The transaction will be signed in another function.

File: src/components/pegout/RskWalletConnection.vue [hunk 4]
```diff
@@ -28,23 +28,33 @@
...
+  @Action(constants.WEB3_SESSION_CLEAR_ACCOUNT, { namespace: 'web3Session' }) clearAccount !: () => void;
+
   @Emit()
-  // eslint-disable-next-line class-methods-use-this
   connectWallet() {
-    console.log('connecting to wallet');
+    this.connectWeb3()
+      .then(() => {
+        this.$emit('openAddressDialog');
+      });
```
Comment (ronaldsg20) [inline src/components/pegout/RskWalletConnection.vue:53]: We are using the decorator @Emit('event') syntax.

**Review by ronaldsg20 — CHANGES_REQUESTED** [review]

File: src/components/pegout/AddressDialog.vue (hunk 2 above)
Comment (alexjavabraz) [reply src/components/pegout/AddressDialog.vue:46]: solved

File: src/components/pegout/AddressDialog.vue (hunk 1 above)
Comment (alexjavabraz) [reply src/components/pegout/AddressDialog.vue:11]: solved.

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by ronaldsg20 — APPROVED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**